web/modules/custom/username/src/Controller/UserNameController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Drupal\username\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * [Description UserNameController]
+ */
+class UserNameController extends ControllerBase {
+
+  /**
+   * @var [type]
+   */
```

Comment:
This is not proper documentation.

Reply:
NO, actually I was trying something and thought to provide the doc block at the end , then I forgot. 
Added the doc.